Cannot change internal keys

If I try to change authorization keys, for example those:

```
whisk:
  ingress:
     type: Standard
     domain: sciabarra.cloud
     apiHostName: sciabarra.cloud
     apiHostPort: 443
  auth:
    system: "965279e9-cdaf-4e1e-8a2f-5f49da805e4d:AVmLjhb3jlgPmBh69l0jt2Gn871Rqx79uNeDp8mM1d7tgwUkX1HanK7Msq0OVL5e"
    guest: "965279e9-cdaf-4e1e-8a2f-5f49da805e4d:AVmLjhb3jlgPmBh69l0jt2Gn871Rqx79uNeDp8mM1d7tgwUkX1HanK7Msq0OVL5e"
```

When I deploy with helm and I check the status with kubectl, I get multiple Error for the `install-packages-xxx` and I cannot set the key with `wsk property set`.

If I leave the default values, removing whisk.auth.* everything works fine.

The error message from openwhisk is lousy (ie, non-existent), but you can't use the same auth for multiple namespaces.  If you look at the controller logs, you'll see a pile of stack traces like the one below that correspond to the `wsk` calls from the install-packages job.
```

[2019-01-23T16:57:44.532Z] [ERROR] [#tid_4rZD6fGbFGoHhe0SrA7BJaUYvvxr44t6] [Identity] subjects/identities[965279e9-cdaf-4e1e-8a2f-5f49da805e4d] is not unique
[2019-01-23T16:57:44.532Z] [INFO] [#tid_4rZD6fGbFGoHhe0SrA7BJaUYvvxr44t6] [Identity] invalidating CacheKey(965279e9-cdaf-4e1e-8a2f-5f49da805e4d)
[2019-01-23T16:57:44.532Z] [ERROR] [#tid_4rZD6fGbFGoHhe0SrA7BJaUYvvxr44t6] [BasicAuthenticationDirective] authentication error: java.lang.IllegalStateException: uuid is not unique
[2019-01-23T16:57:44.543Z] [ERROR] Error during processing of request: 'uuid is not unique'. Completing with 500 Internal Server Error response. To change default exception handling behavior, provide a custom ExceptionHandler.
java.lang.IllegalStateException: uuid is not unique
        at org.apache.openwhisk.core.entity.Identity$.$anonfun$get$8(Identity.scala:111)
        at scala.util.Success.$anonfun$map$1(Try.scala:251)
        at scala.util.Success.map(Try.scala:209)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
        at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
        at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
        at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
        at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
        at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)

```

If I use two unique non-default auth keys in mycluster.yaml, then the install works just fine.

--- a/helm/openwhisk/configMapFiles/initCouchDB/initdb.sh
+++ b/helm/openwhisk/configMapFiles/initCouchDB/initdb.sh
@@ -7,10 +7,18 @@ pushd /openwhisk
     git checkout $OW_GIT_TAG_OPENWHISK
 popd
 
-# Copy the secrets whisk.auth.guest and whisk.auth.system into the cloned tree
-# overwriting the default values we cloned from git
+# Install the secrets whisk.auth.guest and whisk.auth.system into the cloned tree
+# after removing the defaults inherited from the checkout of openwhisk
+rm -f /openwhisk/ansible/files/auth.guest /openwhisk/ansible/files/auth.whisk.system
 cp -f /etc/whisk-auth/guest /openwhisk/ansible/files/auth.guest
 cp -f /etc/whisk-auth/system /openwhisk/ansible/files/auth.whisk.system
+
+# Sanity check: all subjects must have unique keys
+if cmp -s /openwhisk/ansible/files/auth.guest /openwhisk/ansible/files/auth.whisk.system; then
+    echo "FATAL ERROR: unable to initialize the OpenWhisk subjects database."
+    echo "Cannot use identical keys for whisk.auth.system and whisk.auth.guest."
+    exit 1
+fi
 
 # generate db_local.ini so the ansible jobs know how to access the database
 pushd /openwhisk/ansible

--- a/helm/openwhisk/configMapFiles/installPackages/myTask.sh
+++ b/helm/openwhisk/configMapFiles/installPackages/myTask.sh
@@ -13,6 +13,7 @@ export PROVIDER_DB_URL=$PROVIDER_DB_PROTOCOL://$PROVIDER_DB_USERNAME:$PROVIDER_D
 git clone https://github.com/apache/incubator-openwhisk openwhisk
 pushd openwhisk
     git checkout $OW_GIT_TAG_OPENWHISK
+    rm -f /openwhisk/ansible/files/auth.guest /openwhisk/ansible/files/auth.whisk.system
 popd
 
 # Setup env for installRouteMgmt.sh

